### PR TITLE
Feature vlm super

### PR DESCRIPTION
**Pull request by planes**
# Pull Request Checklist
 * Merge in the latest Develop branch changes to your branch
 * Remove .pyc files from your repository:
   * Linux/Mac
     * find . -name *.pyc -delete
   * Windows
     * del /S *.pyc
 * Run automatic regression and makes sure everything is passing
 * Did you make a new regression test that covers your new code?
 * Did you update your docstrings?
 * Did you update your headers to include your name and date?
 * Do a final compare with the Develop branch to be sure what you're changing

### CODE REVIEW

**Comment by planes** [comment]
I need to change the VLM to reduce the memory usage for appveyor.

**Review by timdmacdo — COMMENTED** [review]
Just a few minor comments here.

File: regression/scripts/aerodynamics/sears_haack.py [hunk 1]
```diff
@@ -115,7 +115,7 @@ def main():
     polar.drag = CD        
     
     # load older results
-    #save_results(polar)
+    save_results(polar)
```
[the remark this answers is not in the document]
Comment (planes) [reply regression/scripts/aerodynamics/sears_haack.py]: Yeah, I saw this earlier and fixed but hadn't pushed.

File: trunk/SUAVE/Components/Energy/Converters/Propeller.py [hunk 2]
```diff
@@ -57,7 +57,7 @@ def __defaults__(self):
         self.hub_radius                = 0.0
         self.twist_distribution        = 0.0
         self.chord_distribution        = 0.0
-        self.mid_chord_aligment        = 0.0
+        self.mid_chord_alignment       = 0.0
```
[the remark this answers is not in the document]
Comment (planes) [reply trunk/SUAVE/Components/Energy/Converters/Propeller.py:60]: Matthew had included them originally. It's a small enough change that it probably doesn't warrant it's own PR.

File: trunk/SUAVE/Input_Output/OpenVSP/mach_slices.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+except ImportError:
+    # This allows SUAVE to build without OpenVSP
+    pass
+import numpy as np
+
+# ----------------------------------------------------------------------
+#  Mach Slices
+# ----------------------------------------------------------------------
+
+def mach_slices(vehicle,mach,angle_of_attack=[0.],number_slices = 100):
```
[the remark this answers is not in the document]
Comment (planes) [reply trunk/SUAVE/Input_Output/OpenVSP/mach_slices.py]: I meant to keep this in another branch (super_corrections). I wanted to do a separate PR for this and the other boom scripts. Removed.

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/compute_wing_induced_velocity.py [hunk 4]
```diff
@@ -14,82 +13,79 @@
...
+    YC    = np.atleast_2d(VD.YC*1.) 
+    ZC    = np.atleast_2d(VD.ZC*1.)  
+    XA_TE = np.atleast_2d(VD.XA_TE*1.)  
+    XB_TE = np.atleast_2d(VD.XB_TE*1.)  
+    ZA_TE = np.atleast_2d(VD.ZA_TE*1.)  
+    ZB_TE = np.atleast_2d(VD.ZB_TE*1.)      
+    
+    # supersonic corrections
+    beta_2     = 1-mach**2
+    beta_2_1d  = beta_2*1.
```
Comment (rachealerhard) [inline trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/compute_wing_induced_velocity.py:78]: doesn't look like beta_2_1d is used anywhere

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/generate_propeller_wake_distribution.py [hunk 5]
```diff
@@ -68,13 +66,13 @@ def generate_propeller_wake_distribution(prop,thrust_angle,m,VD,init_timestep_of
     wake_skew_angle = np.arctan(mu_prop/lambda_tot)
 
     # reshape gamma to find the average between stations 
-    gamma_new = np.zeros((m,Na,(Nr-1)))
-    gamma_new = (gamma[:,:,:-1] + gamma[:,:,1:])*0.5
+    gamma_new = np.zeros((m,Na,(Nr-1))) #[control points, azimuth  , radial stations -1 ] one less because ring
+    gamma_new = (gamma[:,:,:-1] + gamma[:,:,1:])*0.5 #  not sure if correct
```
Comment (planes) [inline trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/generate_propeller_wake_distribution.py:70]: @mclarke2 can you be sure if this is correct?

**Review by rachealerhard — COMMENTED** [review]
Looks good to me, a few small changes

File: trunk/SUAVE/Input_Output/OpenVSP/vsp_read_wing.py [hunk 6]
```diff
@@ -181,7 +183,7 @@ def vsp_read_wing(wing_id, units_type='SI'):
 			if i < segment_num:      # This excludes the tip xsec, but we need a segment in SUAVE to store airfoil.
 				sweep     = vsp.GetParmVal(wing_id, 'Sweep', 'XSec_' + str(i)) * Units.deg
 				sweep_loc = vsp.GetParmVal(wing_id, 'Sweep_Location', 'XSec_' + str(i))
-				AR        = vsp.GetParmVal(wing_id, 'Aspect', 'XSec_' + str(i))
+				AR        = 2*vsp.GetParmVal(wing_id, 'Aspect', 'XSec_' + str(i))
```
[the remark this answers is not in the document]
Comment (rachealerhard) [reply trunk/SUAVE/Input_Output/OpenVSP/vsp_read_wing.py:186]: Actually this looks to be accounted for with the x_sec_1_span definition in line 133

[the remark this answers is not in the document]
Comment (planes) [reply trunk/SUAVE/Input_Output/OpenVSP/vsp_read_wing.py:186]: Yeah, that's also not the aspect ratio of a full wing. It's the aspect ratio of a segment of a wing.

File: trunk/SUAVE/Analyses/Aerodynamics/Vortex_Lattice.py [hunk 7]
```diff
@@ -108,7 +115,7 @@ def __defaults__(self):
         
         self.evaluate                                = None
         
-    def initialize(self,use_surrogate,n_sw,n_cw,propeller_wake_model,ito,wdt):
+    def initialize(self,use_surrogate,n_sw,n_cw,propeller_wake_model,ito,wdt,nwts):
```
[the remark this answers is not in the document]
Comment (planes) [reply trunk/SUAVE/Analyses/Aerodynamics/Vortex_Lattice.py:118]: Done

File: trunk/SUAVE/Input_Output/OpenVSP/mach_slices.py [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+    Properties Used:
+    N/A
+    """       
+    
+    # Write the vehicle
+    write(vehicle,vehicle.tag,write_file=False)
+    
+    # Calculate the mach angle and adjust for AoA
+    mach_angle = np.arcsin(1/mach[0])
+    roty = mach_angle - angle_of_attack[0]
```
[the remark this answers is not in the document]
Comment (planes) [reply trunk/SUAVE/Input_Output/OpenVSP/mach_slices.py]: File deleted

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/compute_wing_induced_velocity.py (hunk 4 above)
Comment (planes) [reply trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/compute_wing_induced_velocity.py]: Good catch. lol I deleted that right before you commented.

File: trunk/SUAVE/Components/Energy/Converters/Propeller.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (mclarke2) [reply trunk/SUAVE/Components/Energy/Converters/Propeller.py:60]: Second Emilio's response

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/generate_propeller_wake_distribution.py (hunk 5 above)
Comment (mclarke2) [reply trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/generate_propeller_wake_distribution.py:70]: I averaged the circulations along the propeller blade to reduce the dimension by 1 so its an approximation. The reason for this is that I use the radial stations to define the distributed vortex elements, i,e, if they are 20 stations, they are 19 vortex rings spanwise. The other alternative that I looked into was increasing the discretization of the propeller to 21 allowing me to retain the correct dimension of gamma - this lead to an issue in the slipstream contraction function which also reduced the dimension of the wake by 1 i.e. 19 - as a result, I opted to do an average between stations.

**Review by timdmacdo — APPROVED** [review]

**Pull request merged**